File: database/generate_password.php
<?php
// Script para generar la contraseña hash correcta
$password = 'admin123';
$hash = password_hash($password, PASSWORD_DEFAULT);

echo "Contraseña: " . $password . "\n";
echo "Hash: " . $hash . "\n";

// Verificar que funciona
if (password_verify($password, $hash)) {
    echo "✓ Hash válido\n";
} else {
    echo "✗ Hash inválido\n";
}
?>

File: database/generate_password.py
#!/usr/bin/env python3
import bcrypt
import hashlib

# Generar hash de la contraseña admin123
password = b'admin123'
salt = bcrypt.gensalt()
hash_result = bcrypt.hashpw(password, salt)

print("Contraseña: admin123")
print("Hash generado:", hash_result.decode('utf-8'))

# Verificar que funciona
if bcrypt.checkpw(password, hash_result):
    print("✓ Hash válido")
else:
    print("✗ Hash inválido")

# También generar con PHP compatible
import hashlib
import base64

# Simular PHP password_hash con bcrypt
php_hash = bcrypt.hashpw(password, bcrypt.gensalt(12))
print("Hash PHP compatible:", php_hash.decode('utf-8'))